annotator-core/src/main/java/edu/ucr/cs/riple/core/Config.java

```diff
@@ -525,21 +523,19 @@ public void initializeAdapter(FieldDeclarationStore fieldDeclarationStore) {
     Path serializationVersionPath = target.dir.resolve("serialization_version.txt");
     if (!serializationVersionPath.toFile().exists()) {
       // Older versions of NullAway.
-      this.adapter = new NullAwayV0Adapter(this, fieldDeclarationStore);
-      return;
+      throw new RuntimeException(
+          "This annotator version does not support using version of NullAway, please upgrade NullAway or use version 1.3.4 of Annotator.");
```

Comment:
Can we specify _which_ versions of NullAway are (un)supported?
```suggestion
          "This annotator version requires NullAway 0.10.6 or higher (using SerializeFixMetadataVersion=2), please upgrade NullAway or use version 1.3.4 of Annotator.");
```